### PR TITLE
fix some issues

### PR DESCRIPTION
**Pull request by vasya539**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
change logo size, remove green border for inputs, destroy case with else, change placeholder when file is chosen, confirm for deletion every where

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
image

**Comment by vasya539** [comment]
image

File: vendor/assets/javascripts/show_file_name_in_file_field.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  if ($('#file')[0].files[0].name.length > 15) {
+    // var file_name = $('#file')[0].files[0].name;
+    var file_name = $('#file')[0].files[0].name.split("").slice(0, 15).join("") + "...";
+  }
+  else {
+    var file_name = $('#file')[0].files[0].name;
+  }
+  $('span.file-custom').text(file_name);
+  $('span.file-custom').addClass("file-custom-file-chosen");
+})
```
Comment (igorkasyanchuk) [inline vendor/assets/javascripts/show_file_name_in_file_field.js]: крапка з комою має бути в кінці

і ще тут не гуд що воно не в колбеці, має бути jquery load or turbolinks load 
коли ти вже можеш звертись до $

**Pull request merged**